### PR TITLE
Host platform constants cleanups

### PR DESCRIPTION
**Pull request by dabrahams**
Use sites looked overly verbose with `HostPlatform.` qualification so I changed it to `Host.` in the final commit.  We can of course roll that commit back.

### CODE REVIEW

File: Sources/Utils/Host.swift [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+
+  /// The name of the environment variable containing the executable search path.
+  public static let pathEnvironmentVariable = operatingSystem == .windows ? "Path" : "PATH"
+
+  /// The separator between elements of the environment's executable search path.
+  public static let pathEnvironmentSeparator: Character = operatingSystem == .windows ? ";" : ":"
+
+  /// The file extension applied to binary executables.
+  public static let executableSuffix = operatingSystem == .windows ? ".exe" : ""
+}
```
Comment (kyouko-taiga) [inline Sources/Utils/Host.swift:28]:
```suggestion

}
```

File: Sources/Utils/Host.swift [hunk 2]
```diff
@@ -0,0 +1,28 @@
...
+  #if os(macOS)
+  /// The operating system.
+  public static let operatingSystem: OperatingSystem = .macOS
+  #elseif os(Linux)
+  /// The operating system.
+  public static let operatingSystem: OperatingSystem = .linux
+  #elseif os(Windows)
+  /// The operating system.
+  public static let operatingSystem: OperatingSystem = .windows
+  #endif
```
Comment (kyouko-taiga) [inline Sources/Utils/Host.swift:18]: Did you run swift-format? IIRC it will indent the contents of `#if ... #endif`.

**Review by kyouko-taiga — APPROVED** [review]
LGTM.

File: Sources/Utils/Host.swift (hunk 2 above)
Comment (dabrahams) [reply Sources/Utils/Host.swift:18]: Obviously I did not!  Thanks for the reminders

**Pull request merged**